acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/VoluntaryExitAcceptanceTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.test.acceptance;
+
+import java.io.File;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import tech.pegasys.teku.bls.BLSKeyPair;
+import tech.pegasys.teku.test.acceptance.dsl.AcceptanceTestBase;
+import tech.pegasys.teku.test.acceptance.dsl.BesuNode;
+import tech.pegasys.teku.test.acceptance.dsl.TekuNode;
+import tech.pegasys.teku.test.acceptance.dsl.TekuValidatorNode;
+import tech.pegasys.teku.test.acceptance.dsl.TekuVoluntaryExit;
+
+public class VoluntaryExitAcceptanceTest extends AcceptanceTestBase {
+
+  @Test
+  void shouldChangeValidatorStatusAfterSubmittingVoluntaryExit() throws Exception {
+    final BesuNode eth1Node = createBesuNode();
+    eth1Node.start();
+
+    final List<BLSKeyPair> validatorKeys = createKeysAndSendDeposits(eth1Node, 4);
+
+    final File validatorInfoTar = createValidatorKeystores(validatorKeys, "keys", "passwords");
```

Comment:
I think this should be encapsulated by the DSL better.  I'd say `createKeysAndSendDeposits` should be pushed into `TekuDepositSender` rather than the test base class, and it then returns a custom `ValidatorKeys` object.  So this becomes something like:
```
final ValidatorKeys validatorKeys = createTekuDepositSender().sendValidatorDeposits(eth1Node, 4);
```

and then instead of `withValidatorKeys("/opt/teku/keys:/opt/teku/passwords")`,  both the validator node and the validator exit node would just have `withValidatorKeys(validatorKeys)` and `ValidatorKeys` then has a method to get the keys as a tarball (which it can cache to avoid creating the keystores twice) and the copying into docker just becomes a part of that then.

Reply:
Makes sense. Done.